pkg/config/process.go

```diff
@@ -35,8 +48,9 @@ func setupProcesses(config Config) {
 	config.SetKnown("process_config.dd_agent_env")
 	config.SetKnown("process_config.enabled")
 	config.SetKnown("process_config.intervals.process_realtime")
-	config.SetKnown("process_config.queue_size")
-	config.SetKnown("process_config.rt_queue_size")
+	procBindEnvAndSetDefault(config, "process_config.queue_size", DefaultCheckQueueSize)
+	procBindEnvAndSetDefault(config, "process_config.process_queue_bytes", DefaultProcessQueueBytes)
+	procBindEnvAndSetDefault(config, "process_config.rt_queue_size", DefaultRTCheckQueueSize)
 	config.SetKnown("process_config.max_per_message")
 	config.SetKnown("process_config.max_ctr_procs_per_message")
 	config.SetKnown("process_config.cmd_port")
```

Comment:
Can we move `max_per_message`, `max_ctr_procs_per_message`, `cmd_port` and `expvar_port` as well?

Reply:
I'll create a follow up PR to move these settings. The idea of the current PR is to focus on the queue settings and validate the approach that we're taking to migrate settings that need validation.